src/DssSpell.sol

```diff
@@ -73,189 +53,105 @@ contract DssSpellAction is DssAction {
-        DssExecLib.setChangelogVersion("1.9.2");
-    }
+        // ----------- Maker Open Market Commitee Proposal -----------
+        // https://vote.makerdao.com/polling/QmVG38FK?network=mainnet#poll-detail
 
-    function integrateCentrifugeCollateral(CentrifugeCollateralValues memory collateral) internal {
-        address MIP21_LIQUIDATION_ORACLE =
-            DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+        // ETH-B Stability Fee Decrease 6% to 5%
+        DssExecLib.setIlkStabilityFee("ETH-B", FIVE_PCT, true);
 
-        address vat = DssExecLib.vat();
+        // Maximum Debt Ceiling Decreases.
+        DssExecLib.setIlkAutoLineDebtCeiling("LRC-A", 1 * MILLION); // Decrease 3 million to 1 million.
 
-        // Sanity checks
-        require(GemJoinAbstract(collateral.MCD_JOIN).vat() == vat, "join-vat-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).ilk() == collateral.ilk, "join-ilk-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).gem() == collateral.GEM, "join-gem-not-match");
-        require(GemJoinAbstract(collateral.MCD_JOIN).dec() == DSTokenAbstract(collateral.GEM).decimals(), "join-dec-not-match");
+        DssExecLib.removeIlkFromAutoLine("UNIV2ETHUSDT-A");  // Decrease 10 million to zero.
+        (,,,uint256 univ2EthUsdtLine,) = VatAbstract(MCD_VAT).ilks("UNIV2ETHUSDT-A");
+        DssExecLib.setIlkDebtCeiling("UNIV2ETHUSDT-A", 0); // -univ2EthUsdtLine
 
-        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
-            collateral.ilk, collateral.PRICE, collateral.DOC, collateral.TAU
-        );
-        (,address pip,,) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(collateral.ilk);
+        DssExecLib.removeIlkFromAutoLine("UNIV2DAIUSDT-A");  // Decrease 10 million to zero.
+        (,,,uint256 univ2DaiUsdtLine,) = VatAbstract(MCD_VAT).ilks("UNIV2DAIUSDT-A");
+        DssExecLib.setIlkDebtCeiling("UNIV2DAIUSDT-A", 0); // -univ2DaiUsdtLine
 
-        // Set price feed for RWA-00x
-        DssExecLib.setContract(DssExecLib.spotter(), collateral.ilk, "pip", pip);
+        uint256 removed = add(univ2EthUsdtLine, univ2DaiUsdtLine);
 
-        // Init RWA-00x in Vat
-        Initializable(vat).init(collateral.ilk);
-        // Init RWA-00x in Jug
-        Initializable(DssExecLib.jug()).init(collateral.ilk);
+        uint256 Line = VatAbstract(MCD_VAT).Line();
+        VatAbstract(MCD_VAT).file("Line", sub(Line, removed));
 
-        // Allow RWA-00x Join to modify Vat registry
-        DssExecLib.authorize(vat, collateral.MCD_JOIN);
+        // -----------  Increase UNIV2DAUUSDC-A Maximum Debt Ceiling -----------
+        // https://vote.makerdao.com/polling/QmRAuMND?network=mainnet#poll-detail
 
-        // Set ilk/global DC
-        DssExecLib.increaseIlkDebtCeiling(collateral.ilk, collateral.CEIL, true);
+        DssExecLib.setIlkAutoLineParameters("UNIV2DAIUSDC-A", 250 * MILLION, 10 * MILLION, 8 hours); // 50 million to 250 million.
```

Comment:
Is the comment correct here?

Reply:
As far as I can tell yes.
See here - https://vote.makerdao.com/polling/QmRAuMND?network=mainnet#poll-detail
We increase the max ceiling to 250M.